src/main/drivers/dshot.h

```diff
@@ -57,16 +57,15 @@ typedef struct dshotTelemetryQuality_s {
 extern dshotTelemetryQuality_t dshotTelemetryQuality[MAX_SUPPORTED_MOTORS];
 #endif // USE_DSHOT_TELEMETRY_STATS
 
-#define DSHOT_NORMAL_TELEMETRY_MASK                 (1 << DSHOT_TELEMETRY_TYPE_eRPM)
-#define DSHOT_EXTENDED_TELEMETRY_MASK               (~DSHOT_NORMAL_TELEMETRY_MASK)
-#define DSHOT_TELEMETRY_RANGE_MASK                  (0x0f00u)
+#define DSHOT_TELEMETRY_RANGE_MASK                  0x0e00u   // bits 9-11. bit 8 is always 0 for telemetry
+#define DSHOT_TELEMETRY_RANGE_SHIFT                 9u
```

Comment:
```suggestion
#define DSHOT_TELEMETRY_RANGE_MASK                  (0x0e00u)   // bits 9-11. bit 8 is always 0 for telemetry
#define DSHOT_TELEMETRY_RANGE_SHIFT                 (9u)
```

Reply:
I was asked to remove braces for "simple constants", see https://github.com/betaflight/betaflight/pull/13855#discussion_r1734781616

Reply:
Ah sorry have missed that.

Reply:
Technically, its mentioned in https://betaflight.com/docs/development/codingstyle#implementation
Most code does not use this kind of parentheses in macros, but they are necessary in all nontrivial cases.